week0002/hienhpss/python/week0002.py

```diff
@@ -0,0 +1,37 @@
+def spi_round(n=4,m=4,x=0):
+	'''Looping through the spiral path to return the next
+	cell'''
+	if (x>=(n+1)/2) or (x>=(m+1)/2):
+		return
+	for i in range(x,m-x-1):
+		yield (x,i)
+	for i in range(x,n-x-1):
+		yield(i,m-x-1)
+	for i in range(m-x-1,x,-1):
+		yield(n-x-1,i)
+	for i in range(n-x-1,x,-1):
+		yield(i,x)
```

Comment:
Avoid generators for now :) Why complicating the issue when you can just use loops? 🍡